js/exporter/jspdf/pdf_table.js

```diff
@@ -34,6 +53,34 @@ export class PdfTable {
                     currentCell.drawTopBorder = false;
                 }
             }
+
+
+            if(!cells[i].skip) {
+                if(!isDefined(cells[i].rect)) {
+                    cells[i].rect = {};
```

Comment:
Let's mark this member as 'for internal use': cells[i]._rect = {};

Reply:
ok